### PR TITLE
RI-36 install django-reversion

### PR DESCRIPTION
**Pull request by dpoirier**
This will log all changes to the models where it's enabled,
including the user, timestamp, and the actual data changed.

### CODE REVIEW

File: conf/salt/django_reversion.sls [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Anywhere we're running Django with django-reversion, we need to
+# run "manage.py createinitialrevisions" after each deploy
```
Comment (vkurup) [inline conf/salt/django_reversion.sls]: Let's add this to the README, so the developer sets up their environment correctly.

**Comment by vkurup** [comment]
Are we going to have any directory structure to the items we stick in `conf/salt/`? I'm fine with the way you have it, just curious.

Wondering to myself why we didn't use something like this for Libya...

:cake: after the README update.

**Comment by dpoirier** [comment]
@vkurup was this what you had in mind? https://github.com/theirc/rescue_id/commit/8402809b2080ef8829da17f02ec4b8a31cade2c4

**Comment by vkurup** [comment]
That's perfect.

**Pull request merged**